src/DbTarget.php

```diff
@@ -9,13 +9,10 @@
 use Yiisoft\Db\Connection\ConnectionInterface;
 use Yiisoft\Log\Target;
 
-use function microtime;
-
 /**
- * `DbTarget` stores log messages in a database table.
+ * Stores log messages in a database table.
  *
- * Database schema could be initialized by applying migration:
- * {@see \Yiisoft\Log\Target\Db\Migration\M202101052207CreateLog}.
+ * Database schema can be initialized, you must apply the migration of the SQL file of the implementation to use.
```

Comment:
```suggestion
 * To initialize database schema import SQL file for the corresponding database.
```

I'd add `@see` reference to helpers there.